core/src/main/java/com/sequenceiq/cloudbreak/job/stackpatcher/config/StackPatchConfig.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.cloudbreak.job.stackpatcher.config;
+
+import java.util.Map;
+
+import com.sequenceiq.cloudbreak.domain.stack.StackPatchType;
+
+public abstract class StackPatchConfig {
+
+    private final Map<String, String> config;
+
+    public StackPatchConfig(ExistingStackPatcherConfig existingStackPatcherConfig, StackPatchType stackPatchType) {
+        Map<StackPatchType, ExistingStackPatcherConfigPatch> patchConfigs = existingStackPatcherConfig.getPatchConfigs();
```

Comment:
Create test for this logic. And a generic one in terms of all child config classes in this PR: create tests for them as well to see they are returning the right value from the config map.